cli/compose/loader/loader.go

```diff
@@ -808,28 +808,58 @@ var transformStringList TransformerFunc = func(data any) (any, error) {
 	}
 }
 
-func transformMappingOrListFunc(sep string, allowNil bool) TransformerFunc {
-	return func(data any) (any, error) {
-		return transformMappingOrList(data, sep, allowNil), nil
-	}
-}
+var transformHostsList TransformerFunc = func(data any) (any, error) {
+	// Transform pairs of host and IP addr that may be represented as a map, or a list
```

Comment:
This description could be moved to be a proper docstring for the `transformListOrMapping` func below for better IDE integration

Reply:
Done.